moto/ecs/models.py

```diff
@@ -315,7 +335,7 @@ def create_from_cloudformation_json(
 
         ecs_backend = ecs_backends[region_name]
         return ecs_backend.create_service(
-            cluster, service_name, task_definition, desired_count
+            cluster, service_name, desired_count, task_definition
```

Comment:
Please used named variables here, to avoid any confusion next time we have to switch the order